index.html

```diff
@@ -59,6 +74,9 @@
       fill-opacity: .125;
       shape-rendering: crispEdges;
     }
+    .overview {
```

Comment:
Here also better to use `.timeline .overview` or something of that nature to prevent clashes with other classes on the cbioportal page.